src/main/scala/esmeta/cfg/Func.scala

```diff
@@ -64,6 +64,10 @@ case class Func(
   lazy val isMethod: Boolean =
     irFunc.kind == FuncKind.ConcMeth || irFunc.kind == FuncKind.InternalMeth
 
+  /** check whether it is closure */
+  lazy val isCloure: Boolean =
```

Comment:
```suggestion
  lazy val isClosure: Boolean =
```

Reply:
Good catch! fixed in [4468439](https://github.com/es-meta/esmeta/pull/159/commits/4468439781e817c683a385cf7ad2466c5a705bc3)